src/reports/citytax/report.ts

```diff
@@ -0,0 +1,102 @@
+import {BerlinCityTaxRowItemModel, CityTax} from "./interfaces";
+import {addDays, alert, formattedExecutionTime, getDates, lodash} from "shared";
+import {getAccountTransactions, getReservations} from "api/data";
+import {BookingModels, FinanceModels} from "api/schema";
+
+
+export function generateCityTaxReport(city: string, property: string, startDate: string, endDate: string) {
+    const cityTax = CityTax[city as keyof typeof CityTax];
+    if (cityTax == CityTax.HAMBURG) {
+        alert('Hamburg city tax report will be implemented soon!')
+        return;
+    }
+    const transactionStartDate = addDays(startDate, -7);
+    const transactionEndDate = addDays(endDate, 7);
+    const transactions = getTransactions(property, transactionStartDate, transactionEndDate, startDate, endDate);
+    const reservations = getReservations(property, transactionStartDate, transactionEndDate);
+
+    const sheet = createSheetWithReportInfo(city, property, endDate, startDate);
+
+    switch (cityTax) {
+        case CityTax.BERLIN:
+            generateBerlinCityTax(sheet, transactions, reservations);
+            break;
+    }
+}
+
+export function getCities() {
+    return Object.keys(CityTax).filter((item) => {
+        return isNaN(Number(item));
```

Comment:
looks like you need these values, you can create some `Dictionary` or use `String Enum` to avoid of creating numeric keys 

```typescript
export enum CityTax { BERLIN = 'BERLIN', HAMBURG = 'BERLIN'}
```